code/style.css

```diff
@@ -1,4 +1,309 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/** HEADER **/
+
+header {
+  padding: 10px 20px;
+  background: #FFFFFF;
+  color: #121212;
+  font-family: Helvetica;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  header {
+    justify-content: space-evenly;
+  }
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  /*to spread the lines, since they are stacked*/
+  justify-content: space-between;
+  height: 12px;
+  width: 18px;
+  margin: 0px, 0px, 0px, -80px;
+}
+
+.hamburger div {
+  width: 18px;
+  height: 2px;
+  background-color: #121212;
+}
+
+@media (min-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+}
+
+header img {
+  width: 130px;
+  height: 30.42px;
+}
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  .header-image {
+    display: none;
+  }
+}
+
+/** NAVIGATION BAR**/
+nav {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  nav {
+    display: flex;
+    align-items: center;
+  }
+}
+
+@media (min-width: 1024px) {
+  .nav-image {
+    display: none;
+  }
+  nav a:nth-last-of-type(3) {
+    display: none;
+    /*had to remove <a>; there was space left if only <img> was removed*/
+  }
+}
+
+nav a {
+  color: #121212;
+  text-decoration: none;
+  margin: 0px 12px;
+}
+
+main {
+  padding: 0px 20px;
+}
+
+@media (min-width: 1024px) {
+  main {
+    padding: 0px 30px;
+  }
+}
+
+/** BIG NEWS BOX**/
+
+.big-news {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.big-news img {
+  width: 100%;
+  height: 200px;
+}
+
+@media (min-width: 768px) {
+  .big-news img {
+    width: 45%;
+    height: 90%;
```

Comment:
We shouldn't manipulate with both `width` and `height` - your image is stretched because of that